frontend/src/lib/components/DateFilter/DateFilter.tsx

```diff
@@ -101,7 +101,7 @@ export function DateFilter({
                     const dateValue = dateFilterToText(
                         values[0],
                         values[1],
-                        'No date selected',
+                        'Date range specified in the respective Insight',
```

Comment:
The tweaks around autocapture look good (I pushed a couple more though, so let me know if that makes sense), but I don't _think_ this solves the issue with the dashboard time range selector being confusing. There, the problem seems to be "Custom" being shown, and not "No date selected" (which I actually don't know how to get to show up anywhere)

Reply:
Cool, thanks for the changes! Didn't want to change the "Custom" label here as it looks like a more complicated change. Will do this in a follow up PR.

Reply:
https://github.com/PostHog/posthog/pull/16870

Reply:
It definitely. I really don't know where this "No date selected"/"Date range specified in the respective Insight" was shown at all, do you have any pointers how to check this out?

Reply:
Ah, got you. This is the tooltip for the "Custom" option in the dashboard date range selector.
<img width="434" alt="Screenshot 2023-08-02 at 01 38 43" src="https://github.com/PostHog/posthog/assets/1851359/3aa8fe58-18ad-401b-b451-aadebe9443f9">

Reply:
Ohh, I see, didn't notice  it before